custom/nic_compliance/utils.py

```diff
@@ -66,35 +72,41 @@ def verify_password(password, encoded_password):
     return PASSWORD_HASHER.verify(password, encoded_password)
 
 
-def obfuscated_passwords_redis_key_for_user(username):
-    return REDIS_LOGIN_ATTEMPTS_LIST_PREFIX + username
+def obfuscated_password_redis_key_for_user(username, obfuscated_password):
+    return '_'.join([REDIS_LOGIN_ATTEMPTS_LIST_PREFIX, username, obfuscated_password])
 
 
-def get_obfuscated_passwords(username):
+def get_obfuscated_passwords(username, obfuscated_password):
     client = get_redis_client()
-    return client.get(obfuscated_passwords_redis_key_for_user(username), [])
+    return client.get(obfuscated_password_redis_key_for_user(username, obfuscated_password), [])
```

Comment:
should be `client.get(...., False)`

Reply:
Dont need the method anymore, removing it.